nixos/modules/virtualisation/qemu-vm.nix

```
@@ -474,6 +474,11 @@ in
               type = types.path;
               description = lib.mdDoc "The mount point of the directory inside the virtual machine";
             };
+            options.securityModel = mkOption {
+              type = types.enum [ "passthrough" "mapped" "mapped-xattr" "mapped-file" "none" ];
```

```suggestion
              type = types.enum [ "passthrough" "mapped-xattr" "mapped-file" "none" ];
```
You can skip `mapped`, it seems to be an alias for backward compatibility.